src/components/VtToast.vue

```diff
@@ -19,7 +19,8 @@
                      @vt-finished="finish" />
         <div class="vt-content">
             <h2 v-if="status.title" class="vt-title" v-text="status.title" />
-            <p v-if="status.body" class="vt-paragraph" v-html="status.body" />
+            <div v-if="isJSXBody" class="vt-paragraph"><Node :node="status.body"/></div>
```

Comment:
should we just remove the vt-paragraph and only render the node when given JSX? :thinking:

Reply:
Also, should we add jsx support at the VtIcon.vue component too?

Reply:
I think that the vt-paragraph is fine there - it just applies a font-size, some margin, and a color for the theme. I kept it so that more simple jsx bodies are still homogenous with the normal ones. The main reason I switched it to a `div` is to ensure valid dom-nesting as `p` has a limited set of allowed children.

Reply:
> Also, should we add jsx support at the VtIcon.vue component too?

I was considering that as well, I think it makes sense to. The main thing is that it is really dangerous to use user input of any kind in `v-html`